assets/airports/saez.json

```diff
@@ -806,55 +878,105 @@
             "icao": "ATOVO2A",
             "name": "Atovo Two Alpha",
             "rwy": {
-                "11": ["EZE12", "_EZE10DME43", "_EZE10DME48", "_EZE10DME53", "_EZE10DME58", "_EZE10DME63", "_EZE10DME68"],
-                "29": ["EZE11", "_EZE10DME85", "_EZE10DME80", "_EZE10DME75"],
-                "17": ["EZE13", "_EZE10DME62", "_EZE10DME67"],
-                "35": ["EZE10", "_EZE10DME98", "_EZE10DME93", "_EZE10DME88", "_EZE10DME85", "_EZE10DME80", "_EZE10DME75"]
+                "11": ["EZE12", "_EZE126010", "_EZE141010", "_EZE156010", "_EZE171010", "_EZE186010", "_EZE201010"],
+                "29": ["EZE11", "_EZE251010", "_EZE236010", "_EZE221010"],
+                "17": ["_EZE182004", "_EZE183010", "_EZE198010"],
+                "35": ["EZE10", "_EZE291010", "_EZE275010", "_EZE260010", "_EZE251010", "_EZE236010", "_EZE221010"]
```

Comment:
At the time (way back when I first added the airport), I created the DME arc using GPSVisualizer's range ring calculator, which uses 3 degree increments I think. While I agree 3 degrees makes for a lot of fixes, I'd say 20 degrees is too far apart and It won't look like aircraft are really following a DME arc. I'd probably use something like 5-10 degrees.